fonts/generate-smufl-code.py

```diff
@@ -26,62 +27,58 @@ def get_file_content(filepath):
 
 def write_file_content(filepath, content):
     location = os.path.realpath(os.path.dirname(__file__))
-    file = open(os.path.join(location, filepath), 'w')
+    file = open(os.path.join(location, filepath), "w")
     file.write(content)
     file.close()
 
-# Retrieve all elements with given tag name from xml
-
-
-def get_elements(xml, tag):
-    root = ET.fromstring(bytes(xml, encoding='utf-8'))
-    return root.findall('.//' + tag)  # XPath, recursive
-
 #########################
 #  Parse supported.xsl  #
 #########################
 
 # Retrieve dictionary mapping glyph code to smufl name
 
 
-def get_supported_glyph_codes():
-    xml = get_file_content('supported.xsl')
-    glyphs = get_elements(xml, 'glyph')
-    result = {}
+def get_supported_glyph_codes() -> Dict:
+    """Retrieve dictionary with supported SMuFL codepoints and name."""
+    supported_xsl = ET.parse("supported.xsl")
+    glyphs = supported_xsl.findall(".//glyph")
+    supported_glyphs = {}
     for glyph in glyphs:
-        result[glyph.attrib["glyph-code"]] = glyph.attrib["smufl-name"]
-    return result
+        supported_glyphs[glyph.attrib["glyph-code"]
+                         ] = glyph.attrib["smufl-name"]
```

Comment:
This is a strange line break, before the closing square bracket...

Reply:
This comes from autopep-8